quic/s2n-quic-core/src/recovery/bbr.rs

```diff
@@ -429,9 +429,25 @@ impl CongestionController for BbrCongestionController {
         }
     }
 
+    //= https://www.rfc-editor.org/rfc/rfc8899#section-3
+    //= type=exception
+    //= reason=See https://github.com/aws/s2n-quic/issues/959
+    //# An update to the PLPMTU (or MPS) MUST NOT increase the congestion
+    //# window measured in bytes [RFC4821].
+
+    //= https://www.rfc-editor.org/rfc/rfc9002#section-7.2
+    //= type=exception
+    //= reason=The maximum datagram size remains at the minimum (1200 bytes) during the handshake
+    //# If the maximum datagram size is decreased in order to complete the
+    //# handshake, the congestion window SHOULD be set to the new initial
+    //# congestion window.
```

Comment:
i am not sure how the handshake and a decrease in datagram size relates here.

maybe stating that we are operating thruput on a packet basis rather than bytes basis is maybe more relevant.

Reply:
The RFC is saying that if the MTU is decreased during the handshake, then you should revert the congestion window to the initial window. But since we never decrease the MTU during the handshake, we put in an exception